### PR TITLE
U4X-728: Delete unused tasks from UgandaEMR

### PR DESCRIPTION
**Pull request by semujju**
https://metsprogramme.atlassian.net/browse/U4X-728?atlOrigin=eyJpIjoiMTk1MWI1YzA3ZjZmNGM2MWFlODY4N2ZjNDM5MjhlMTEiLCJwIjoiaiJ9

### CODE REVIEW

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -339,27 +339,6 @@
         </insert>
     </changeSet>
 
-
```
Comment (slubwama) [inline api/src/main/resources/liquibase.xml:342]: Besides deleting these in liquibase have you considered also removing the classes which support the task

**Review by slubwama — CHANGES_REQUESTED** [review]
Is this the only task to delete in sync

File: api/src/main/resources/liquibase.xml (hunk 1 above)
Comment (semujju) [reply api/src/main/resources/liquibase.xml:342]: There are no classes related to those tasks

Comment (semujju) [reply api/src/main/resources/liquibase.xml:342]: @slubwama thats the only task t delete in sync

**Review by slubwama — APPROVED** [review]

**Pull request merged**